src/testdir/test_startup.vim

```diff
@@ -537,3 +537,12 @@ func Test_zzz_startinsert()
   endif
   call delete('Xtestout')
 endfunc
+
+func Test_issue3969()
+  if has('gui_running')
+    " Can't catch the output of gvim.
+    return
+  endif
+  let out = system(GetVimCommand() . ' --clean -es -X -V1 -c "echon ''hello''" -cq')
```

Comment:
This result `out` is actually `not found in 'runtimepath': "ftdetect/*.vim" hello` so has not matched the expected value strictly.
Need append `-u NONE` (or remove `--clean` since `GetVimCommand()` includes `-u NONE`), and can replace `assert_match()` by `assert_equal()`.